reports/011/reproreview.Rmd

```diff
@@ -0,0 +1,165 @@
+
+# Reviewed paper
+
+Zourlidou, S., Golze, J., and Sester, M.: Traffic Regulation Recognition using Crowd-Sensed GPS and Map Data: a Hybrid Approach, AGILE GIScience Ser., 3, 22, https://doi.org/10.5194/agile-giss-3-22-2022
+
+# Summary
+
+The paper compares several models for traffic control recognition at junctions, which are built upon Random Forest and Gradient Boosting classifiers. The analysis makes use of 2 datasets, which correspond to the cities of Chicago and Hannover. The first dataset was published previously by Ahmed et al. (2014) and was made publicly available for the needs of this study through this link: [https://drive.google.com/file/d/1GHtPpyoziF3hMV_LtyvTfnIBRQ8BdX_e/edit](https://drive.google.com/file/d/1GHtPpyoziF3hMV_LtyvTfnIBRQ8BdX_e/edit). The data for Hannover were collected by the authors of the paper (as mentioned in Subsection 3.1) and have not been made openly available yet. The code were the analysis was based on, was also declared as intellectual property and thus, it is not publicly shared. However, the authors agreed to share confidentially all their materials with the Reproducibility Committee for the needs of the review. For the needs of this review, we reproduced all the Tables and Figures of *Section 5 - Results* and of the Appendix. The outcome of the review was discussed with the authors to resolve any dissimilarities and eventually the reproduced results were in accordance with the final uploaded version of the manuscript. Therefore, the reproduction of the paper is considered **successful**.
+
+\clearpage
+
+# Reproducibility reviewer notes
+
+The implementation of the analysis is based on a PostgreSQL database with a PostGIS extension and on Jupyter notebooks. The PostgreSQL database holds the data that needs to be accessed and processed during various steps of the computational analysis, which is implemented entirely on Jupyter notebooks. In order to properly recreate the pipeline, detailed instructions about the installation of the necessary software were provided by the authors. The exact versions of the tools/environments that were used are listed below:
+- Windows 10 OS
+- pgAdmin4 6.8
+- PostgreSQL 13.7
+- PostGIS 3.1.4
+- Anaconda Navigator 2.1.1
+- Jupyter Notebook 6.4.11
+- Python 3.9.12
+
+The pipeline of the workflow was not hard to reproduce. The authors took a couple of weeks to organize the materials properly and eventually responded with elaborate instructions that were easy to follow. The notebooks were structured in a sensible way that matches the way the results are presented in the paper and the reproduced figures and numbers were explicitly corresponding with the paper. A list of Python packages was not included in the materials. All the necessary Python packages were installed using Anaconda navigator, unless indicated otherwise by the instructions.
+
+### Reproduced figures
+
+All the reproduced figures are listed below:
```

Comment:
Be clear here if the reproduced figures did (not) match what you saw in the paper, and if meaningful what the differences are.

Reply:
Made it explicit, please check.